README.md

```diff
@@ -23,6 +23,17 @@ npm install @mongodb-js/zstd
 | Linux x64 musl   | ✓      | ✓      | ✓      | ✓      | ✓      |
 | Linux arm64 musl | ✓      | ✓      | ✓      | ✓      | ✓      |
 
+## MongoDB Node.js Driver Version Compatibility
+
+Only the following version combinations with the [MongoDB Node.js Driver](https://github.com/mongodb/node-mongodb-native) are considered stable.
+
+|               | `@mongodb-js/zstd@1.x` |
+| ------------- | ---------------------- |
+| `mongodb@6.x` | ✓ `^1.1.0`           |
+| `mongodb@5.x` | ✓                      |
```

Comment:
```suggestion
| `mongodb@5.x` | ✓ `1.1.0`              |
```

Reply:
@durran Is there something that makes the 1.0.0 version incompatible so that `@1.x` is inaccurate? 5.0.0 was released without any peer dep requirements, and the dev dependency was at 1.0.0, so that's why the compat table in the driver readme also has `^1.0.0` for 5.x.

Reply:
The idea was that we'd update these tables once upon each major version release and then not have to touch them. I think if 5.x worked with 1.0.0 and when we added it to peer dep we made it 1.1.0 instead, then that's a bug in that we broke semver compat, and we should fix the 5.x latest to unbreak the compat like we did for kerberos.

Reply:
Fine with that. Thanks!